package.json

```diff
@@ -37,29 +37,29 @@
     "tool"
   ],
   "dependencies": {
-    "dateformat": "~3.0.3",
+    "dateformat": "~4.6.2",
     "eventemitter2": "~0.4.13",
     "exit": "~0.1.2",
-    "findup-sync": "~0.3.0",
-    "glob": "~7.1.6",
+    "findup-sync": "~5.0.0",
+    "glob": "~8.1.0",
```

Comment:
@vladikoff just FYI this bump could wreak havoc for Windows users since backslashes need to be escaped in v8.x.